package.json

```
@@ -1,6 +1,6 @@
 {
   "private": true,
-  "version": "0.2.2",
+  "version": "0.3.1",
```

Main is version 0.4.0 right now
```suggestion
  "version": "0.4.1",
```